app/grandchallenge/reader_studies/models.py

```diff
@@ -993,9 +1001,7 @@ def calculate_score(self, answer, ground_truth):
         else:
             ans = [answer]
             gt = [ground_truth]
-        return self.SCORING_FUNCTIONS[self.scoring_function](
-            ans, gt, normalize=True
```

Comment:
Removing `normalize=True` is going to be a problem with the existing method. To keep that use a partial:

``` 
ScoringFunction.ACCURACY: partial(accuracy_score, normalize=True)
```

where it is defined above (see https://docs.python.org/3/library/functools.html#functools.partial)

Flipping `ans` and `gt` here was the right thing to do, that was a bug :-)

Reply:
Normalize: default of accuracy score has normalize is True. That's why I thought it could be left out. sklearn.metrics.accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None). Is it then still necessary to add it as a partial?
Bug: yeah I thought so. Of course for accuracy it doesn't effect the result but for auc it does.

Reply:
Ah great, didn't see that and thought that because it was set that it wasn't the default, then indeed it can be left out and this block is fine, thanks!